frontend/app/services/work-package-service.js

```diff
@@ -75,6 +76,47 @@ module.exports = function($http,
   }
 
   var WorkPackageService = {
+    initializeWorkPackage: function(projectIdentifier, initialData) {
+      var changes = _.clone(initialData);
+      if (initialData.type) {
+        changes._links = changes.links || {};
+        changes._links.type = {
+          href: initialData.type
+        };
+        delete changes.type;
```

Comment:
This begs two questions:

* If linked properties like type are misplaced inside `changes` (`type` instead of `_links/type`), what about other linked values (e.g. status and priority)
* Looking at line 82: If we copy `changes.links` into `changes._links` beforehand, why is the type not already part of the changes?